frontend/src/metabase/search/components/SearchSidebar/ToggleSidebarFilter/ToggleSidebarFilter.tsx

```diff
@@ -1,21 +1,50 @@
+/*
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ * TODO: Change mantine switch import before merging!!
```

Comment:
😂 This is one hell of a way to make sure we don't forget to fix this.

Reply:
If it's loud enough I won't forget 😄